src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -5072,6 +5078,7 @@ protected function addRefFkScheduledForDeletionAttribute(&$script, ForeignKey $r
     /**
      * An array of objects scheduled for deletion.
      * @var ObjectCollection|{$className}[]
+     * @phpstan-var  ObjectCollection&\Traversable<{$className}>
```

Comment:
No inline alignment needed
```suggestion
     * @phpstan-var ObjectCollection&\Traversable<{$className}>
```